src/lucky/action_pipes.cr

```diff
@@ -178,4 +182,20 @@ module Lucky::ActionPipes
   def continue : Lucky::ActionPipes::Continue
     Lucky::ActionPipes::Continue.new
   end
+
+  def publish_before_event(pipe_name : String, continued : Bool)
+    Lucky::Events::PipeEvent.publish(
+      name: pipe_name,
+      position: Lucky::Events::PipeEvent::Position::Before,
+      continued: continued
+    )
+  end
+
+  def publish_after_event(pipe_name : String, continued : Bool)
+    Lucky::Events::PipeEvent.publish(
+      name: pipe_name,
+      position: Lucky::Events::PipeEvent::Position::After,
```

Comment:
I tested in a plain crystal class that I could pass `:after` here, but for some reason in this scenario, it throws a no method overload with `position : Symbol` 🤔  It's probably fine though since this isn't really user facing.